src/main/scala/nl/biopet/bioconda/BiocondaUtils.scala

```diff
@@ -0,0 +1,139 @@
+ */
+
+package nl.biopet.bioconda
+
+import com.typesafe.sbt.git.GitRunner
+import nl.biopet.utils.conversions.{any2map, yamlFileToMap}
+import ohnosequences.sbt.GithubRelease.keys.TagName
+import org.kohsuke.github.{GHRelease, GHRepository}
+import sbt.internal.util.ManagedLogger
+import sbt.{File, URL}
+
+import scala.collection.JavaConverters
+import scala.sys.process._
+
+object BiocondaUtils {
+  def getSourceUrl(tag: TagName, repo: GHRepository): URL = {
+    val repoName = repo.getName
+    val repoOwner = repo.getOwnerName
+    val releaseList = repo.listReleases().asList()
+    val releases =
+      JavaConverters.collectionAsScalaIterable(releaseList).toList
+    val currentRelease = releases.find(x => x.getTagName == tag)
+    if (currentRelease.isEmpty) {
+      throw new java.io.FileNotFoundException(
+        s"'$tag' tag not present on release page. Please release on github before publishing to bioconda.")
+    }
+    val assets = JavaConverters
+      .collectionAsScalaIterable(
+        currentRelease.getOrElse(new GHRelease).getAssets)
+      .toList
+    val releaseJar =
+      // Finds all jars. This assumes only one jar is released.
+      assets.find(
+        x =>
+          x.getBrowserDownloadUrl.endsWith(".jar") &&
+            !x.getBrowserDownloadUrl.endsWith("javadoc.jar") &&
+            !x.getBrowserDownloadUrl.endsWith("sources.jar")
+      )
+    releaseJar match {
+      case Some(x) => new URL(x.getBrowserDownloadUrl)
+      case _ =>
+        throw new java.io.FileNotFoundException(
+          s"No valid release jar found for: $tag")
+    }
+  }
+  def branchExists(branch: String,
+                   repo: File,
+                   git: GitRunner,
+                   log: ManagedLogger,
+                   remotes: Boolean = false): Boolean = {
+    // TODO: Find a git command that just returns branches as a list. (Without * in front of the branch you are on)
+
+    // Without "--no-color" scala doesn't match the strings properly. Color matters in string comparison!
+    val branchList: Array[String] = {
+      if (remotes) {
+        git("branch", "-a", "--no-color")(repo, log).split("\\n")
+      } else {
+        git("branch", "--no-color")(repo, log).split("\\n")
+      }
+    }
+    val branches: Array[String] = branchList.flatMap(x => {
+      val branchInfo: Array[String] = { x.replaceFirst("\\*", "").split("/") }
+      if (branchInfo.isEmpty) None
+      else Some(branchInfo.lastOption.getOrElse("").trim())
+    })
+    branches.contains(branch)
+
+  }
+  def getVersionFromYaml(metaYaml: File): String = {
+    val packageValue: Map[String, Any] =
+      any2map(
+        yamlFileToMap(metaYaml)
+          .getOrElse("package", Map()))
+    val version = packageValue.getOrElse(
+      "version",
+      throw new Exception(s"No version found in ${metaYaml.getPath}"))
+    version.toString
+  }
+
+  def dockerInstalled(log: ManagedLogger,
```

Comment:
I have rewritten this method so it can use an environment map, which makes the code a lot cleaner.
Using `Sys` from common-utils does not add much. Instead it takes the extensive logging methods that the standard library has away. Since I am not using async processes, using the standard library here is preferred.